src/imu_vn_100.cpp

```diff
@@ -438,7 +438,13 @@ void ImuVn100::Disconnect() {
 }
 
 void ImuVn100::PublishData(const VnDeviceCompositeData& data) {
-  rclcpp::Time now = this->now();
+  if (!has_time_zero_) {
+    device_time_zero_ = data.timeStartup;
+    ros_time_zero_ = this->now();
```

Comment:
Minor - the order should be swapped here. You want to capture the ROS Time as close to data acquisition as possible: 
```C++
void ImuVn100::PublishData(const VnDeviceCompositeData& data) {
  if (!has_time_zero_) {
    ros_time_zero_ = this->now();          // <-- swapped
    device_time_zero_ = data.timeStartup;  // <-- lines
...
```

Reply:
Yeah, having it closer is *slightly* better, though most of the benefit from this PR is having that offset be constant (basically the time from the first data acquisition to this call), instead of increasing with every call.

I have a couple more upcoming changes to submit, I'll make this minor change as one of those.